Resources/config/messenger.xml

```diff
@@ -28,6 +28,16 @@
             <argument type="service" id="doctrine" />
         </service>
 
+        <!--
+        The following service isn't prefixed by the "doctrine.orm" namespace in order for end-users to just use
+        the "doctrine_open_transaction_logger" shortcut in message buses middleware config
+        -->
+        <service id="messenger.middleware.doctrine_open_transaction_logger" class="Symfony\Bridge\Doctrine\Messenger\DoctrineOpenTransactionLoggerMiddleware" abstract="true" public="false">
+            <argument type="service" id="doctrine" />
+            <argument type="constant">null</argument>
```

Comment:
`type="constant"` with `null` as name looks suspicious to me

Reply:
Indeed. Even if it works, this is not conceptually correct. Pls reomve the type @lyrixx

Reply:
I'm not sure to understand. What should I use?

Reply:
<argument>null</argument>

Reply:
Thanks. fixed.